utils.py

```diff
@@ -42,7 +45,36 @@ def get_redis_cache_data(*keys):
     redis_key = '-'.join(filter(None, list(keys)))
     return cache.get(redis_key)
 
+
 def get_values_list_from_dataclass(data_class):
     if data_class:
-        return [value for value in asdict(data_class).values() if value != UNSET ]
+        return [value for value in asdict(data_class).values() if value != UNSET]
     return []
+
+
+async def get_filtered_ordered_paginated_qs(
+    qs, filters, order, pagination
+):
+    if filters:
+        qs = filter_apply(filters, qs)
+    if order:
+        qs = ordering_apply(order, qs)
+    if pagination:
+        qs = pagination_apply(pagination, qs)
+    return await get_async_list_from_queryset(qs)
+
+
+def get_country_iso3_list():
+    '''
+    Returns existing countires iso3 list, Used to filter countries that should be
+    visiable in dashboard. Also used to ignore conflited area countries.
+    '''
+    from apps.visualization.models import Countries
+
+    cache_key = 'existing-countires-iso3-list'
+    iso3_list = get_redis_cache_data(cache_key)
+    if not iso3_list:
+        iso3_list = Countries.objects.values_list('iso3', flat=True)
+        set_redis_cache_data(cache_key, value=iso3_list)
+        return iso3_list
+    return iso3_list
```

Comment:
Why are we caching this value?
If this is used to filter out in the queryset then just using Countries.objects.values('iso3') in the filter will be faster.